CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
The primary reason for the filename is that Github will show a [little green bar](https://github.com/blog/1184-contributing-guidelines) when making a PR - so the person can just click on on it and tada, see the coding guidelines here and then. Developer experience is very important to me because they're humans too.